src/lib/components/workflow/workflow-advanced-search.svelte

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
+  import { fade } from 'svelte/transition';
+  import { fly } from 'svelte/transition';
+  import { page } from '$app/stores';
+
+  import { updateQueryParameters } from '$lib/utilities/update-query-parameters';
+
+  import Input from '$lib/holocene/input/input.svelte';
+  import Button from '$lib/holocene/button.svelte';
+  import { workflowFilters } from '$lib/stores/filters';
+  import { toListWorkflowFilters } from '$lib/utilities/query/to-list-workflow-filters';
+
+  let manualSearchString = '';
+
+  $: query = $page.url.searchParams.get('query');
+
+  function setManualString(query) {
+    manualSearchString = query;
+  }
+
+  $: {
+    setManualString(query);
+  }
+
+  const onSearch = () => {
+    try {
+      $workflowFilters = toListWorkflowFilters(manualSearchString);
+    } catch (e) {}
+
+    updateQueryParameters({
+      url: $page.url,
+      parameter: 'query',
+      value: manualSearchString,
+      allowEmpty: true,
+    });
+  };
+
+  function handleClearInput() {
```

Comment:
When clearing the search input the filters are removed, but the text remains in the drop down filters.
| With query | After it has been cleared |
|--|--|
|![Screen Shot 2022-10-12 at 10 05 16 AM](https://user-images.githubusercontent.com/15069288/195393364-5c5cb9e8-510e-425a-a24e-7db50bbad29c.png)|![Screen Shot 2022-10-12 at 10 05 32 AM](https://user-images.githubusercontent.com/15069288/195393463-baa3c732-b7b2-4f22-9621-b2250b838006.png)|